docusaurus/docs/dev-docs/backend-customization/controllers.md

```diff
@@ -141,23 +201,22 @@ A specific `GET /hello` [route](/dev-docs/backend-customization/routes) is defin
 <TabItem value="js" label="JavaScript">
 
 ```js "title="./src/api/hello/routes/hello.js"
-
 module.exports = {
   routes: [
     {
-      method: 'GET',
-      path: '/hello',
-      handler: 'hello.index',
-    }
-  ]
-}
+      method: "GET",
```

Comment:
Is it based on the documentation prettier config or smth like that?

Reply:
We don't have any Prettier or other linters config (and I don't want one at the moment, less tooling = easier maintenance + flexibility). I assume this comes from Ben's setup and it's OK for me, no worries :)

Reply:
Yeah, that's why, I think my vs code just uses some default prettier settings on it